docs/UserGuide.md

```diff
@@ -154,6 +155,7 @@ Format: `edit INDEX [r/ROLE] [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* The result of an edit can be identical to the original person, but it **cannot be identical to any other existing user in the address book**.
```

Comment:
I don't think there is a need to specify this line? I don't think its considered a bug when you can edit the person to itself. It doesn't seem to add much value then

Reply:
I think there is no harm in mentioning it as it clarifies the intended behavior of the software.